docs/Videos.md

```diff
@@ -0,0 +1,20 @@
+# Videos
+
+* Find out how to [quickstart][0] using cocoapods and carthage.
```

Comment:
These lines seem to be referring to a different person and not consistent.
Perhaps something like:
```
Find out how to:
* [quickstart][0] using cocoapods and carthage
* [setup][1] the SDK to enable push notification and rich media
* [integrate][2] the SDK, with real code examples
```